spyder/plugins/tests/test_ipythonconsole.py

```diff
@@ -74,9 +74,45 @@ def close_console():
     return console
 
 
+@pytest.fixture
+def ipyconsole_stderr(request):
```

Comment:
@dalthviz, are you sure we need an extra fixture for this? Is it not possible to add an extra parameter to the previous fixture?

Reply:
Yep, you are right :+1: no need of a extra fixture